app/va/va_profile/va_profile_client.py

```diff
@@ -31,6 +31,8 @@
     'sms': 'Text',
 }
 
+# source for valid phone types for SMS delivery:
+# https://docs.aws.amazon.com/pinpoint/latest/developerguide/validate-phone-numbers.html#validate-phone-numbers-example-responses
```

Comment:
This was added because it was being discussed when I spoke with Kyle about this PR. The documentation will be good to have available in the future.